src/extremeweatherbench/metrics.py

```diff
@@ -68,32 +81,42 @@ class MaximumMAE(Metric):
     """Mean absolute error of forecasted maximum values."""
 
     def compute(self, forecast: xr.DataArray, observation: xr.DataArray):
-        max_mae_values = []
-        observation_spatial_mean = observation.mean(["latitude", "longitude"])
-        observation_spatial_mean = observation_spatial_mean.where(
-            observation_spatial_mean.time.dt.hour % 6 == 0, drop=True
-        )
-        forecast_spatial_mean = forecast.mean(["latitude", "longitude"])
-        for init_time in forecast_spatial_mean.init_time:
-            if forecast.name == "air_temperature":
-                max_date = observation_spatial_mean.idxmax("time").values
-                max_value = observation_spatial_mean.sel(time=max_date).values
-                init_forecast_spatial_mean, _ = self.align_datasets(
+        if forecast.name == "air_temperature":
+            max_mae_values = []
+            observation_spatial_mean = observation.mean(["latitude", "longitude"])
+            forecast_spatial_mean = forecast.mean(["latitude", "longitude"])
+            observation_spatial_mean = self._align_observations_temporal_resolution(
+                forecast_spatial_mean, observation_spatial_mean
+            )
+            for init_time in forecast_spatial_mean.init_time:
+                max_datetime = observation_spatial_mean.idxmax("time").values
+                max_value = observation_spatial_mean.sel(time=max_datetime).values
+                init_forecast_spatial_mean, _ = utils.temporal_align_dataarrays(
                     forecast_spatial_mean,
                     observation_spatial_mean,
                     pd.Timestamp(init_time.values).to_pydatetime(),
                 )
 
-                if max_date in init_forecast_spatial_mean.time.values:
-                    lead_time = init_forecast_spatial_mean.where(
-                        init_forecast_spatial_mean.time == max_date, drop=True
+                if max_datetime in init_forecast_spatial_mean.time.values:
+                    # Subset to +-48 hours centered on the maximum temperature timestamp
+                    filtered_forecast = utils.center_forecast_on_time(
+                        init_forecast_spatial_mean,
+                        time=pd.Timestamp(max_datetime),
+                        hours=48,
```

Comment:
Seems like `hours` is a good candidate for a parameter that `RegionalRMSE` should accept.